README.md

```diff
@@ -47,9 +47,9 @@ Server components can be made async and can securely perform server-side logic a
 // server component
 import db from 'some-db';
 
-import { Gallery } from '../components/gallery.js';
+import { Gallery } from '~/components/gallery.js';
```

Comment:
path alias isn't effective at the moment. we need to investigate a way to fix, but not yet done.